src/registry/select/select.spec.tsx

```diff
@@ -62,6 +62,57 @@ test('Switch to multiple sets empty array as default value', async () => {
   expect(submittedComponent.defaultValue).toStrictEqual([]);
 });
 
+test('Switch to multiple sets empty array as default value when initial is null', async () => {
+  const user = userEvent.setup({advanceTimers: jest.advanceTimersByTime});
+  const onSubmit = jest.fn();
+
+  const component = {
+    id: 'wqimsadk',
+    type: 'select',
+    key: 'select',
+    label: 'A select field',
+    dataSrc: 'values',
+    openForms: {
+      dataSrc: 'manual',
+      translations: {},
+    },
+    data: {
+      values: [{value: 'opt1', label: 'Opt1'}],
+    },
+    // @ts-ignore
```

Comment:
```suggestion
    // @ts-expect-error
```
I think this is more appropriate, and I think it'll also give a notice when TS does not error :)

Reply:
Interesting, I read about it https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#ts-ignore-or-ts-expect-error. Although I think I prefer the Python way: using `type: ignore` for everything, and enable the reporting of unnecessary comments in mypy/pyright ;)